### DISCUSSION TITLE
NWBstream environment failing to launch

### DISCUSSION THREAD

**Discussion by ajuavinett** [dandi/helpdesk#55] [General]
### Bug description
NWBstream environment will not launch

#### Expected behaviour
I thought that this would connect to the NWBstream kernel and enable use of the NWB widgets demo, e.g. 

#### Actual behaviour
When trying to connect to the NWBstream environment on the DANDI jupyterhub, I receive the error `Error Starting Kernel: Unhandled error`

### How to reproduce
Use launcher or "select kernel" menu to select the NWBstream kernel.

### Your personal set up
My account on https://hub.dandiarchive.org/ (which has barely been used...), using a Chrome browser.

**Comment by satra** [comment]
The NWBStream kernel is no longer available since PyNWB 2.0 was released with streaming support. This has been merged into the standard environment. You should be able to replace with "Python 3 (ipykernel)".

**Reply by ajuavinett** [reply]
Gotcha, thank you!
